tsl/src/compression/compression.c

```diff
@@ -873,18 +860,11 @@ table_scan_sequence_number(Relation table_rel, int16 seq_num_column_num, ScanKey
  * SEQUENCE_NUM_GAP.
  */
 static int32
-get_sequence_number_for_current_group(Relation table_rel, Oid index_oid,
+get_sequence_number_for_current_group(Relation table_rel, Relation index_rel,
 									  int16 *uncompressed_col_to_compressed_col,
 									  PerColumn *per_column, int n_input_columns,
 									  int16 seq_num_column_num)
 {
-	/* No point scanning an empty relation. */
-	if (table_rel->rd_rel->relpages == 0)
```

Comment:
was this just removed because it was not required or was there another reason?

Reply:
relpages are no longer being updated as we are not updating stats when compressing chunks so can't rely on this anymore.